cellforest/plot/r/plot_exit_point.R

```diff
@@ -0,0 +1,5 @@
+dpi = 150
+width_in = plot_width_px / dpi
+height_in = plot_height_px / dpi
```

Comment:
I do think feeding in pixels might be a little non-intuitive but the kind of extremely minor hurdle users can clear. we'll all be passing around the "here are the magic numbers for producing a poster-ready plot" line anyhow

Reply:
Which other parts are you referring to that use pixels?

Btw, my thought with pixels is that on different computers when cellforest is run, because of different pixel density (DPI), the produced plots will size differently depending on the screen of the machine that you're running on which might cause inconsistencies (e.g., when we make that slider-view thing). So I was thinking that if we keep DPI the same and define in pixels then wherever the plots are produced, they will be the same resolution. Any thoughts?